mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/mapping/SearchMappingBuilder.java

```diff
@@ -172,19 +171,6 @@ public <E> SearchMappingBuilder addEntityType(Class<E> type, String entityName,
 		return this;
 	}
 
-	/**
-	 * Enables or disables multi-tenancy.
-	 * <p>
-	 * If multi-tenancy is enabled, every {@link SearchMapping#createSession() session} will need to be assigned a tenant identifier.
-	 *
-	 * @param multiTenancyEnabled {@code true} to enable multi-tenancy, {@code false} to disable it (the default).
-	 * @return {@code this}, for call chaining.
-	 */
-	public SearchMappingBuilder multiTenancyEnabled(boolean multiTenancyEnabled) {
```

Comment:
Can you also remove the other setters that are now in the configuration context? No need to duplicate all that.

Reply:
While doing this (in that last commit), `StandalonePojoMappingSetupHelper` had to be changed as it was based on the builder. I've added the `withConfigurer` and `appendConfigurer` methods. This affected quite a few test configurations along the way.

Reply:
Can't you hide this? You could still have the same methods as before in the setup helper, having them populate a configurer that is added by default.
See `org.hibernate.search.documentation.testsupport.DocumentationSetupHelper#defaultMappingConfigurer` for an example of what I mean.

Reply:
I'm being annoying here because we'll likely have the same problem in Quarkus: we'll need to expose the configurer to users but also configure stuff in Quarkus itself.

If the problem is only about adding entities, then maybe `addEntity` should remain in the builder and not in the configurer...